bodygroupmanager/derma/cl_viewer.lua

```diff
@@ -0,0 +1,149 @@
+local PLUGIN = PLUGIN or {}
+
+local PANEL = {}
+
+function PANEL:Init()
+    --self:SetText("Bodygroup Manager")
+    self:SetSize(720, 720)
+    self:Center()
+    self:SetBackgroundBlur(true)
+    self:SetDeleteOnClose(true)
+
+    self:MakePopup()
+    self:SetTitle("Bodygroup Manager")
+
+    self.clipboard = self:Add("DButton")
+	self.clipboard:Dock(BOTTOM)
+	self.clipboard:DockMargin(0, 4, 0, 0)
+    self.clipboard:SetText("Save Changes")
+    self.clipboard.DoClick = function()
+        local bodygroups = {}
+        for _, v in pairs(self.bodygroupIndex) do
+            table.insert(bodygroups, v.index, v.value)
+        end
+        netstream.Start("ixBodygroupTableSet", self.target, bodygroups)

```

Comment:
You should convert usage of `netstream` to use the built-in `net` library instead. Otherwise, make it clear in the `README` that you require netstream for the plugin to operate